src/content/community/grants/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Ethereum Foundation & community grant programs
+description: A listing of the grant programs throughout the Ethereum ecosystem.
+lang: en
+---
+
+# Ethereum Grants {#ethereum-grants}
+
+The programs listed below offer a variety of funding grants for projects working to promote the success and growth of the Ethereum ecosystem. Use this as a guide to find and apply for funds to help make your next Ethereum project a success.
+
+### Broader Ecosystem {#broader-ecosystem}
+
+- [EF Ecosystem Support Program](https://esp.ethereum.foundation) - _Rollups, community and education, tooling and testing, infrastructure, interoperability, standards and research, wallets, and more_
+- [Consensys](https://consensys.net/grants/) - _Infrastructure L1 and L2, education & technical knowledge, usability and dev tooling, social impact, security, and more_
+- [MetaCartel](https://www.metacartel.org/) - _Dapp development_
+- [Moloch DAO](https://www.molochdao.com/) - _Privacy, layer 2 scaling, client security, and more_
+- [Open Grants](https://opengrants.com/explore)
+
+### Project Oriented {#project-oriented}
+
```

Comment:
Can we add some explanation under the title. "These projects have their own grants for developing and experimenting with their technology." Or something like that. Project-oriented isn't immediately clear to me.